bibtex-completion.el

```diff
@@ -4,7 +4,7 @@
 ;;         Justin Burkett <justin@burkett.cc>
 ;; Maintainer: Titus von der Malsburg <malsburg@posteo.de>
 ;; URL: https://github.com/tmalsburg/helm-bibtex
-;; Version: 1.0.0
+;; Version: 1.1.0
```

Comment:
Actually, I'm a little unclear how MELPA versioning works.

If it's based on git tags, then since all these packages use the same repo, doesn't this need to be 2.1?

Reply:
Yes, that's my understanding as well (but I might be wrong).